tests/behat/features/bootstrap/SocialDrupalContext.php

```diff
@@ -496,4 +496,23 @@ public function createNodeLikes(TableNode $nodesTable): void {
     }
   }
 
+
+  /**
+   * I wait (seconds) seconds for (field_name) (field_type) be rendered.
+   *
+   * @When /^(?:|I )wait "([^"]*)" seconds for "([^"]*)" "([^"]*)" field be rendered$/
+   */
+  public function iWaitForTheFieldBeRender(string $seconds, string $field_name, string $field_type): void {
```

Comment:
I think what we want to do here, is to not wait for 60 seconds, but use a while loop with a maximum timeout. This loop can check on a given interval (lets say every X00ms), if the field is already rendered. 

This way we don't have to wait for a minute to have the field rendered, if it's there sooner it can already return. If it doesn't find the field after that timeout, we can return an exception. 

That should save us a lot of time